config/bidderinfo.go

```diff
@@ -246,6 +246,35 @@ func processBidderInfos(reader InfoReader, normalizeBidderName func(string) (ope
 			infos[string(normalizedBidderName)] = info
 		}
 	}
+	return processAliasBidderInfo(infos)
+}
+
+func processAliasBidderInfo(infos BidderInfos) (BidderInfos, error) {
```

Comment:
Could you write a unit test specific to `processAliasBidderInfo` where this function is directly called before asserting?

Reply:
I didn't think that is necessary since the unit test for this function was covered in processBidderInfos function unit test. But [added here explicitly for this](https://github.com/prebid/prebid-server/pull/2960/commits/93af4f3201711e7deccfc63b66aa984d01995e30). thanks!